tests/fileVersionTest.js

```diff
@@ -69,32 +61,43 @@ describe('Main: Currently testing file versions management,', function () {
     })
   }
 
-  const fileVersionPath = (fileId, versionId) => MatchersV3.regex(
-   `.*\\/remote\\.php\\/dav\\/meta\\/${fileId}\\/v\\/${versionId}$`,
-   `/remote.php/dav/meta/${fileId}/v/${versionId}`
-  )
-
   describe('file versions of non existing file', () => {
     it('retrieves file versions of not existing file', async function () {
       const provider = createProvider()
-      await getCapabilitiesInteraction(provider)
-      await getCurrentUserInformationInteraction(provider)
+      await getCapabilitiesInteraction(provider, testUser, testUserPassword)
+      await getCurrentUserInformationInteraction(provider, testUser, testUserPassword)
       await provider
-        .uponReceiving(`as '${username}', a PROPFIND request to get file versions of non existent file`)
-        .withRequest(propfindFileVersionsRequestData)
+        .given('the user is recreated', {
+          username: testUser,
+          password: testUserPassword
+        })
+        .given('set random versionId')
+        .uponReceiving(`as '${testUser}', a PROPFIND request to get file versions of non existent file`)
+        .withRequest({
+          method: 'PROPFIND',
+          path: MatchersV3.regex(
+            '.*\\/remote\\.php\\/dav\\/meta\\/42\\/v', /* eslint-disable-line no-template-curly-in-string */
```

Comment:
`/* eslint-disable-line no-template-curly-in-string */` did you forget to remove?